src/main/java/org/opentripplanner/routing/edgetype/StreetEdgeTraversal.java

```diff
@@ -129,6 +145,92 @@ public static StateEditor doTraverse(StreetEdge edge, State s0, RoutingRequest o
             time *= 0.001;
         }
 
+        StateEditor s1 = state.edit(edge);
+        s1.setBackMode(traverseMode);
+        s1.setBackWalkingBike(walkingBike);
+
+        ///////////////////////////////////
+        // NIH weighting preferences
+        OptionSet extraOptions = edge.getExtraOptionFields();
+        NumericFieldSet numericFieldSet = edge.getExtraNumericFields();
+        boolean isAudited = (extraOptions != null) || (numericFieldSet != null);
```

Comment:
There's a bunch of places you have extra parans in this block. AFAIK they aren't necessary, if that's the case it would be best to remove them.